src/dotnet-scaffolding/Microsoft.DotNet.Scaffolding.Internal/Telemetry/TelemetryConstants.cs

```diff
@@ -0,0 +1,13 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Microsoft.DotNet.Scaffolding.Internal.Telemetry;
+
+internal static class TelemetryConstants
+{
+    public static readonly string TELEMETRY_OPTOUT = "DOTNET_SCAFFOLD_TELEMETRY_OPTOUT";
```

Comment:
Do we need to discuss whether there should be a single opt out for all first party scaffolders and the main tool, or if you should be able to opt out individual tools individually? The first time use sentinel is done per-tool. But currently the opt out and the skip are done globally.

Reply:
single opt-out for both is fine I think.